guides/release/tutorial/8.md

```diff
@@ -0,0 +1,159 @@
+We accomplished a lot in the previous chapter! Let's recap &mdash; we started with a blank canvas, added a few pages of content, styled everything to look pretty, dropped in a picture of Tomster, added links between our pages and amazingly, everything worked together flawlessly!
+
+But do we _really_ know that everything is actually working? Sure, we clicked around a bit to confirm that things look as expected. But do we feel confident that we checked _every_ page after the most recent change that we made?
+
+After all, most of us have experienced (or heard horror stories about) making a Small Tweak™ in one area of the app and inadvertently breaking _everything else_ when we weren't looking.
+
+Maybe we can write a checklist somewhere of all the things to check after making changes to our site. But surely, this will get out of hand quickly as we add more features to our app? It is also going to get old really quickly &mdash; repetitive tasks like that are best left to robots.
```

Comment:
```suggestion
Maybe we can write a checklist somewhere of all the things to check after making changes to our site. But surely, this will get out of hand as we add more features to our app. It is also going to get old really quickly &mdash; repetitive tasks like that are best left to robots.
```